src/js/dataMapping/search/tableSearchFields.js

```diff
@@ -176,11 +151,9 @@ const tableSearchFields = {
             'recipient_name',
```

Comment:
You will also need to add some logic to handle sorting by a column in one tab, then switching to a different tab where that column does not exist.